test/e2e/customconfigs/managejobswithoutqueuename_test.go

```diff
@@ -212,11 +212,6 @@ var _ = ginkgo.Describe("ManageJobsWithoutQueueName", ginkgo.Ordered, func() {
 					g.Expect(createdJob.Labels).ShouldNot(gomega.HaveKeyWithValue(controllerconstants.QueueLabel, "default"))
 				}, util.Timeout, util.Interval).Should(gomega.Succeed())
 			})
-			ginkgo.By("setting feature gates back to its original state", func() {
```

Comment:
Also dropping this call because it is already in the afterEach: https://github.com/kubernetes-sigs/kueue/pull/7925/files#diff-7fb50105d6ea57d298c2788bec48c38c69a071f60834733bec81814c93e8bf4cR108-R112